modules/passkey/contracts/4337/SafeECDSASignerLaunchpad.sol

```diff
@@ -12,90 +12,129 @@ import {ERC1271} from "../libraries/ERC1271.sol";
 
 /**
  * @title Safe Launchpad for Custom ECDSA Signing Schemes.
- * @dev A launchpad account implementation that enables the creation of Safes that use custom ECDSA
- * signing schemes that require additional contract deployments over ERC-4337.
+ * @dev A launchpad account implementation that enables the creation of Safes that use custom ECDSA signing schemes that
+ * require additional contract deployments over ERC-4337. Note that it is not safe to rely on this launchpad for
+ * deploying Safes that has an initial threshold greater than 1. This is because the first user operation (which can
+ * freely change the owner structure) will only ever require a single signature to execute, so effectively the initial
+ * owner will always have ultimate control over the Safe during the first user operation and can undo any changes to the
+ * `threshold` during the `setup` phase.
  * @custom:security-contact bounty@safe.global
  */
 contract SafeECDSASignerLaunchpad is IAccount, SafeStorage {
-    bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH = keccak256("EIP712Domain(uint256 chainId,address verifyingContract)");
+    /**
+     * @notice The EIP-712 type-hash for the domain separator used for verifying Safe initialization signatures.
+     * @custom:computed-as keccak256("EIP712Domain(uint256 chainId,address verifyingContract)")
+     */
+    bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH = 0x47e79534a245952e8b16893a336b85a3d9ea9fa8c573f3d803afb92a79469218;
 
-    // keccak256("SafeSignerLaunchpad.initHash") - 1
-    uint256 private constant INIT_HASH_SLOT = 0x1d2f0b9dbb6ed3f829c9614e6c5d2ea2285238801394dc57e8500e0e306d8f80;
+    /**
+     * @notice The storage slot used for the initialization hash of account.
+     * @custom:computed-as keccak256("SafeSignerLaunchpad.initHash") - 1
+     * @dev This value is intentionally computed to be a hash -1 as a precaution to avoid any potential issues from
```

Comment:
Subjective-nit:

```suggestion
     * @dev This value is intentionally computed to be a hash - 1 as a precaution to avoid any potential issues from
```